### ISSUE TITLE
About Loss function.

### ISSUE THREAD

**Issue by AndrewTalYeah** [facebookresearch/swav#117]
In the paper, use L(zt, zs) = l(zt, qs) + l(zs, qt) for training, if only use L=l(zt, qt) for training, that is, one branch of the two data augmentations is removed. Do you think this can train normally?

**Comment by GSusan** [comment]
您好！我已收到邮件，会尽快回复。
